version.txt

```diff
@@ -1 +1 @@
-4.0.0-beta.2
+84.0.0-beta.1
```

Comment:
v83 is already on its stabilization branch and will ship sometimes next week. As far as I understand, 84 is what's on the master branch.

Reply:
Can't wait to test this in production!